includes/adapter/RecipientFromWCOrder.php

```diff
@@ -67,10 +67,24 @@ private function getAddressFromOrder(WC_Order $order, string $type): array
         $street       = WCX_Order::get_meta($order, "_{$type}_street_name") ?: null;
         $number       = WCX_Order::get_meta($order, "_{$type}_house_number") ?: null;
         $numberSuffix = WCX_Order::get_meta($order, "_{$type}_house_number_suffix") ?: null;
-        $addressLine1 = $this->separateStreet($order->{"get_{$type}_address_1"}(), $order, $type);
         $addressLine2 = $order->{"get_{$type}_address_2"}();
+        $isNL         = $order->{"get_{$type}_country"}() === AbstractConsignment::CC_NL;
```

Comment:
Yoda en kan die `$order->{"get_{$type}_country"}` meteen in een var gecached worden? <- geen dealbreaker voor mij